### PR TITLE
Feature book component

### PR DESCRIPTION
**Pull request by akshay-zagade**
- Enquiry form added

### CODE REVIEW

**Review by sachinwakle — COMMENTED** [review]
Please check comments. Let's discuss!

File: src/components/EnquiryDialouge.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+          />
+        </DialogContent>
+        <DialogActions>
+          <Button
+              variant="contained"
+              color="secondary"
+            >
+              Submit Enquiry
+            </Button>
+        </DialogActions>
```
[the remark this answers is not in the document]
Comment (akshay-zagade) [reply src/components/EnquiryDialouge.js:133]: Enclosed form within `<form>` tag

File: src/components/EnquiryDialouge.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+          <TextField
+            id="outlined-basic"
+            label="How can we help you ?"
+            variant="outlined"
+            fullWidth
+            multiline
+            rows={3}
+          />
+        </DialogContent>
+        <DialogActions>
```
[the remark this answers is not in the document]
Comment (akshay-zagade) [reply src/components/EnquiryDialouge.js:126]: Removed `<DialogActions>`

File: src/components/Photos.js [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import React from 'react'
+import EnquiryForm from "./EnquiryForm";
 
 function Photos() {
     return (
         <div>
+            <EnquiryForm />
```
[the remark this answers is not in the document]
Comment (akshay-zagade) [reply src/components/Photos.js:7]: As per our discussion. Enquiry/Book will be always visible to user.

**Review by sachinwakle — APPROVED** [review]
looks fine

**Pull request merged**